package.json

```diff
@@ -1,29 +1,32 @@
 {
   "name": "orbit-db-test-utils",
-  "version": "0.8.4",
+  "version": "0.8.5",
   "description": "",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/orbitdb/orbit-db-test-utils"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "TEST=all nyc mocha"
   },
   "bin": {
     "redis-server": "./bin/redis-server"
   },
+  "standard": {
+    "env": "mocha"
+  },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "babel-loader": "^8.0.6",
```

Comment:
Not sure if we need babel at this level, or if we need to include it so that `orbit-db-test-utils` can be included

Reply:
Not refactoring now